config/src/base.rs

```diff
@@ -46,3 +53,22 @@ impl Default for BaseConfig {
         }
     }
 }
+
+impl BaseConfig {
+    pub fn from_yaml_file(relative_path: &str) -> Result<Self, Report> {
+        let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        path.push(relative_path);
```

Comment:
@zeevmoney I am thinking about changing this so the configuration would be taken from a constant path like:
`~/.dwallet/dwallet_config/eth_config.yml`